# Genetic Distance Network

Phylogenetic relationships based on genetic distance are extremely useful to identify groups and clusters of taxa.
We can visualize phylogenetic relationships with networks using igraph. 


The potential cutoff for your analysis can be determined by identifying a threshold that best differentiates a bimodal distribution of the genetic distances that are typically present in your sequences [MicroTrace Documentation](https://raw.githubusercontent.com/CDCgov/MicrobeTrace/master/docs/MicrobeTrace_Manual.pdf).  Example: The frequency distribution of genetic distances representing genetic distances from known pathogen transmission cases, and the red boxes representing genetic distances from cases without evidence of pathogen transmission.

Variant calling sequences (VCFs) 
|Genomo reference | Lineage|
|--|--|
|PR-102_v3 | NA1 |
|PR-15-019 | EU1 |

`$PATH` to folders `/dfs/Grunwald_Lab/lab_collabs/diagnostic_assays/genome_data/`
